library/src/doo/coverage.clj

```diff
@@ -0,0 +1,57 @@
+(ns doo.coverage
+  (:require [meta-merge.core :refer [meta-merge]]))
+
+(defn- package->preprocessors
+  "Adds preprocessors for sources from a given packages"
+  [->out-dir package]
+  (let [package-path (-> (name package)
+                         (clojure.string/replace #"\." "/"))
```

Comment:
Another replacement needed here is from `-` to `_`. E.g. if your package is `my-app`, the generated source will be in `out-dir/my_app/` and not in `out-dir/my-app/`.